apps/admin_panel/assets/src/omg-user-current/currentUserProvider.js

```diff
@@ -17,7 +17,7 @@ class UserProvider extends Component {
   componentDidMount = async () => {
     if (this.props.currentUserLoadingStatus === 'DEFAULT') {
       const result = await this.props.getCurrentUser()
-      if (!result.data.success) {
+      if (!result.data) {
```

Comment:
Why the change? Shouldn't you be checking the error code as well? Or maybe it's displayed after the push?

Reply:
This is a data provider component, we don't need to handle any error code here.
if the returned result has data, it is guaranteed to be a success action. if it is error, it will return `result.error`